### PR TITLE
fix: Only show status label in queue if transaction is pending

### PR DESCRIPTION
**Pull request by usame-algan**
## What it solves

Resolves [Issue in QA Board](https://www.notion.so/safe-global/Tx-status-is-displayed-as-a-text-in-addition-to-n-of-M-5202a7c1373c4f5cb30fa42e2882cc30)

## How this PR fixes it

- Adds previous condition back to only render `TxStatusLabel` if the transaction is either not in the queue or if it is pending

## How to test it

1. Queue a transaction
2. Observe no status label in the queue
3. Execute the transaction
4. Observe a processing status label
5. Go to the history after execution
6. Observe a Success status label

## Screenshots
Screenshot 2024-04-08 at 14 43 34
Screenshot 2024-04-08 at 14 44 08
Screenshot 2024-04-08 at 14 44 22

Screenshot 2024-04-08 at 14 43 42

## Checklist
* [ ] I've tested the branch on mobile 📱
* [ ] I've documented how it affects the analytics (if at all) 📊
* [ ] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

File: src/components/transactions/TxSummary/index.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ const TxSummary = ({ item, isGrouped }: TxSummaryProps): ReactElement => {
       )}
 
       <Box gridArea="status" justifyContent="flex-end" display="flex" mr={5}>
-        <TxStatusLabel tx={tx} />
+        {(!isQueue || isPending) && <TxStatusLabel tx={tx} />}
       </Box>
```
Comment (katspaugh) [inline src/components/transactions/TxSummary/index.tsx:80]: Shouldn't this whole block be toggled?

Comment (usame-algan) [reply src/components/transactions/TxSummary/index.tsx]: Hiding the whole block also works, thanks!

**Review by katspaugh — APPROVED** [review]
👍

**Comment by francovenica** [comment]
LGTM

**Pull request merged**